docs/_includes/sidebar.html

```diff
@@ -63,6 +60,18 @@
   </ul>
   <!--End (temporary) beta sections-->
 
+  <ul class="sidebar-section">
+    <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/deploying">
+      <li class="title">{{ site.t[page.lang].deploying }}</li>
```

Comment:
We should make this section header a conditional until there's a translated version

Reply:
Added commit cef89ab to only display the **Deployments** section when there is content (including hiding the section header). Currently, this will hide the **Deployments** section for the Japanese translate.

@shaydewael my Jekyll/Liquid skills 🔪 are a little rusty. Would you mind looking at my implementation? If you have a more elegant way to do it, I'd love to see it!